mongodbatlas/data_source_mongodbatlas_clusters.go

```diff
@@ -287,8 +287,9 @@ func dataSourceMongoDBAtlasClusters() *schema.Resource {
 							Computed: true,
 						},
 						"labels": {
-							Type:     schema.TypeSet,
-							Computed: true,
+							Type:       schema.TypeSet,
+							Computed:   true,
+							Deprecated: fmt.Sprintf(DeprecationByDateWithReplacement, "November 2023", "tags"),
```

Comment:
it might be useful to have constants with deprecation dates so it's easier to see next deprecation dates

Reply:
This could be helpful, but I believe configuring due date within jira tickets will be our best bet to make sure we are aware of removing deprecated fields.